modules/jaeger-config-ingester.adoc

```diff
@@ -6,57 +6,50 @@ This REFERENCE module included in the following assemblies:
 [id="jaeger-config-ingester_{context}"]
 = Jaeger Ingester configuration options
 
-Ingester is a service that reads from Kafka topic and writes to another storage backend (Elasticsearch).  If you are using the `allInOne` or `production` deployment strategies, you do not need to configure the Ingester service.
-
+Ingester is a service that reads from a Kafka topic and writes to another storage backend (Elasticsearch).  If you are using the `allInOne` or `production` deployment strategies, you do not need to configure the Ingester service.
 
 .Jaeger Ingester parameters
 [options="header"]
-[cols="l, a, a, a"]
+[cols="l, a, a"]
 |===
-|Parameter |Description |Values |Default value
+|Parameter |Description |Values
 |spec:
   strategy: streaming
   ingester:
     options: {}
 |Configuration options that define the Ingester service.
 |
-|
 
-|ingester:
- options:
-  kafka:
-    consumer:
-      topic: jaeger-spans
+|autoscale:
+|This parameter controls whether to enable/disable autoscaling for the Ingester.  Set to `false` to explicitly disable autoscaling.
```

Comment:
Just a nitpick here, but it might be helpful to word this so it's clear autoscale is true by default.